amplify/backend/backend-config.json

```diff
@@ -8,37 +8,49 @@
             "UserPoolId"
           ],
           "category": "auth",
-          "resourceName": "fitchvocel5266d294"
+          "resourceName": "vocelappcf1453cb"
         },
         {
           "attributes": [
             "Arn",
             "Name"
           ],
           "category": "function",
-          "resourceName": "AdminQueriesfd66d56a"
+          "resourceName": "AdminQueries5b1ab4e5"
```

Comment:
once again, I hope we have instructions for how to update these when moving to a different AWS instance